packaging/hudi-spark-bundle/pom.xml

```diff
@@ -367,7 +369,18 @@
       <artifactId>curator-recipes</artifactId>
       <version>${zk-curator.version}</version>
     </dependency>
-
+<!--    import json to fix the crash when sync meta in spark-->
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20200518</version>
```

Comment:
define the version and dependency in parent pom?

Reply:
done